### PR TITLE
APIs of project Start & End

### PR DESCRIPTION
**Pull request by iamhassanali**

### CODE REVIEW

File: app/controllers/api/v1/admin/project.js [hunk 1]
```diff
@@ -2,6 +2,12 @@ const { ProjectStatus } = global.appEnums;
 const moment = require('moment');
 const { sumBy, flatMap, map, filter } = require("lodash");
 const { asyncMiddleware } = global;
+const {
+  sumBy,
+  flatMap,
+  map,
+  filter
```
Comment (isajjadali) [inline app/controllers/api/v1/admin/project.js]: Where are we using these methods?

File: app/controllers/api/v1/admin/project.js [hunk 2]
```diff
@@ -2,6 +2,12 @@ const { ProjectStatus } = global.appEnums;
 const moment = require('moment');
 const { sumBy, flatMap, map, filter } = require("lodash");
```
Comment (isajjadali) [inline app/controllers/api/v1/admin/project.js:3]: why we require lodash 2 times in a same file?

**Review by isajjadali — CHANGES_REQUESTED** [review]

File: app/controllers/api/v1/admin/project.js (hunk 2 above)
Comment (iamhassanali) [reply app/controllers/api/v1/admin/project.js:3]: It was included by mistake while resolving the conflicts.

File: app/controllers/api/v1/admin/project.js (hunk 1 above)
Comment (iamhassanali) [reply app/controllers/api/v1/admin/project.js]: In some APIs, we are using these methods.

**Pull request merged**